### ISSUE TITLE
 BaseSettings has moved to a new package pydantic-settings

### ISSUE THREAD

**Issue by shivupa** [MolSSI/QCElemental#312]
**Describe the bug**
https://github.com/MolSSI/QCElemental/blob/7b048906d173a5ad872a7185c76c42a16f2136db/qcelemental/util/autodocs.py#L6

`BaseSettings` has moved to a new package `pydantic-settings`

https://docs.pydantic.dev/latest/migration/#basesettings-has-moved-to-pydantic-settings
https://github.com/pydantic/pydantic-settings

**To Reproduce
~conda list~ Not used.
Can't currently run:
`python -c "import qcelemental as qcel; print(qcel.__file__, qcel.__version__)"`

**Expected behavior**
Be able to import qcelemental

**Comment by hjjvandam** [comment]
I am hitting the same problem. The issue seems to be that with Pydantic V2 some [API breaking changes](https://docs.pydantic.dev/2.0/migration/#basesettings-has-moved-to-pydantic-settings) have been introduced. Nevertheless, QCElemental lists this  https://github.com/MolSSI/QCElemental/blob/7b048906d173a5ad872a7185c76c42a16f2136db/pyproject.toml#L34 as a dependency, which includes version 2 and higher. A quick fix would be to restrict pydantic to versions less than 2.

**Comment by loriab** [comment]
Agreed, I need to add the pydantic<2.0 restriction generally and mint a new version. I did do a new build number in conda with the restriction, but that doesn't help the src and pip usages. Thanks for the ping.

**Comment by wiederm** [comment]
Has this been resolved in the current conda build?

**Comment by loriab** [comment]
The conda package v0.26.0 as of an hour ago works in the presence of pydantic v1 or v2. It's still using the v1 API, so I'll leave this open for the v2 migration and for general info.

**Comment by wiederm** [comment]
great, thank you for the fix!
